src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl

```diff
@@ -67,21 +68,37 @@ Return a grid with `GridFittedBottom` immersed boundary.
 Computes ib.bottom_height and wraps in an array.
 """
 function ImmersedBoundaryGrid(grid, ib::AbstractGridFittedBottom)
-    bottom_field = Field{Center, Center, Nothing}(grid)
+    bottom_field = Field((Center, Center, Nothing), grid)
     set!(bottom_field, ib.bottom_height)
     fill_halo_regions!(bottom_field)
     offset_bottom_array = dropdims(bottom_field.data, dims=3)
 
-    # TODO: maybe clean this up
-    IB = typeof(ib).name.wrapper
-    new_ib = IB(offset_bottom_array)
+    new_ib = getname(ib)(offset_bottom_array)
```

Comment:
I think we should keep the `TODO` comment because I'm not sure this is a robust pattern

Reply:
There may be a more julian way to write this